src/views/components/driverCard.js

```diff
@@ -11,6 +12,13 @@ function Driver({ user, dataTestID }) {
         setShowChatModal(false);
     };
 
+    const { banUserResult, handleBanUser, banUserError } = useBanUser();
+
+    const handleBanSubmit = async (event) => {
+        event.preventDefault();
+        await handleBanUser({id:user.ID, status:"ban"});
+    };
+
     return (
         <>
             <Card data-test-id="driver-card">
```

Comment:
You can add `className="mb-3"` to add some padding at the bottom to separate cards a bit